package.json

```diff
@@ -69,7 +69,7 @@
     "react-window-mixins": "0.0.6",
     "scriptjs": "^2.5.8",
     "tether": "^1.1.1",
-    "uncontrollable": "^3.0.2",
+    "uncontrollable": "awseward/uncontrollable.git#reset-prop-on-defaultProp-change",
```

Comment:
Once this has passed testing, https://github.com/procore/uncontrollable/pull/1 should be merged, and this can be changed to
```json
"uncontrollable": "procore/uncontrollable.git",
```